### DISCUSSION TITLE
Is support for .NET packages.props is planned for future releases?

### DISCUSSION THREAD

**Discussion by yuriShafet** [aquasecurity/trivy#5379] [Q&A]
### Question

Is support for .NET packages.props is planned for future releases?
If the answer is no, will PR with this functionality will be something that the team is looking for?

### Target

None

### Scanner

None

### Output Format

None

### Mode

None

**Comment by DmitriyLewen** [comment]
Hello @yuriShafet 
Do you mean [Directory.Packages.props](https://learn.microsoft.com/en-us/nuget/consume-packages/central-package-management)?

**Reply by yuriShafet** [reply]
Yes. But if you but it could also be without "Directory"
https://github.com/dotnet/msbuild/blob/main/eng/Packages.props

**Reply by DmitriyLewen** [reply]
Microsoft docs (https://learn.microsoft.com/en-us/nuget/consume-packages/central-package-management) only say about `Directory.Packages.props`. Have you seen information that “Directory” can be omitted? When is `Directory.Packages.prop` used and when is `Packages.props` used?

**Reply by yuriShafet** [reply]
As could be seen here: https://github.com/microsoft/MSBuildSdks/tree/main/src/CentralPackageVersions
packages.props is an old naming convention of newer Directory.Packages.props.
It still exists in some older projects and has same format as Directory.Packages.props.

**Reply by DmitriyLewen** [reply]
Thanks for explanation!

We think it would be nice to add support for `*Packages.props` files.
Do you have the opportunity to work on this?

**Reply by yuriShafet** [reply]
Sure, I can do that

**Comment by DmitriyLewen** [comment]
I created #5403 for this task.
